### DISCUSSION TITLE
feat: Add black colorScheme to Button component

### DISCUSSION THREAD

**Discussion by Ayushhgupta39** [chakra-ui/chakra-ui#8438] [Ideas]
### Description

I propose adding a new black color variant to the Button component in Chakra UI. This addition would enhance the library's flexibility and usability, allowing developers to create sleek, modern interfaces more easily.

### Problem Statement/Justification

Chakra UI provides a great selection of default button colors. However, one common color that's missing is black. Black buttons are widely used in various user interfaces due to their simplicity and versatility. When developers want to create a black button in Chakra UI, they have to manually customize it, which can be cumbersome and time-consuming.

### Proposed Solution or API

I propose adding a new black color variant to the Button component in Chakra UI.
![image](https://github.com/chakra-ui/chakra-ui/assets/96309609/1193a408-8698-4bbb-b94b-e466f33008b9)

### Alternatives

The alternative solution is the color blackAlpha, which is not exactly the black color developers want to use in many cases. The standard black color has a very high use case, which I believe cannot be replaced by any shade or variant of it.

**Comment by pga5e** [comment]
yes I badly needed it to set a pure black `<Checkbox colorScheme="blackAlpha">
        I Agree with Terms & Conditions!
      </Checkbox>`

and color=black, bgColor=black, colorScheme="black", nothing seemed to work

would great if they could provide black out of the box
